### ISSUE TITLE
Cannot release v4.0.0

### ISSUE THREAD

**Issue by Kuniwak** [Kuniwak/MirrorDiffKit#43]
```console
$ DEVELOPER_DIR=/Applications/Xcode-10.2b.app carthage build --no-skip-current
*** xcodebuild output can be found in /var/folders/q6/l0_p9dsx7nvf8xw__vp1r_4cwtc1_s/T/carthage-xcodebuild.e6s8FF.log
*** Building scheme "MirrorDiffKit-watchOS" in MirrorDiffKit.xcodeproj
Could not find any available simulators for watchOS
```

Related: https://github.com/Carthage/Carthage/issues/2691

**Comment by giginet** [comment]
We've just released Carthage 0.32.0.
Using this, it will fix this issue.

https://github.com/Carthage/Carthage/releases/tag/0.32.0

**Comment by Kuniwak** [comment]
I've published MirrorDiffKit v4.0.0 after upgraded Carthage to 0.32.0.

@giginet Thanks a lot!
